hawc_hal/dec_band_select.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import division
+from builtins import zip
+from past.utils import old_div
+from builtins import object
+import argparse as ap
+import numpy as np
+import scipy.stats       as stats
+import warnings
+import os
+import sys
+import corner as cn
+import healpy            as hp
+import scipy
+
+from pathlib import Path
+
+import uproot
+from threeML.io.logging import setup_logger
+from astropy.coordinates import SkyCoord
+
+with warnings.catch_warnings():
+    import astromodels
+        
+log = setup_logger(__name__)
+log.propagate = False
+
+def dec_index_search(response_file, dec_var, use_module):
```

Comment:
Does this take into account response files in .hd5 format?

Reply:
It seems there is a test with a response file in hd5 format and that's what is breaking the checks.